hermione.ts

```diff
@@ -1,131 +1,3 @@
-                isCliCommandLaunched = true;
-            });
-        });
-    });
-
-    hermione.on(hermione.events.INIT, withMiddleware(async () => {
-        const dbClient = await SqliteClient.create({htmlReporter, reportPath: config.path});
-        const imageStore = new SqliteImageStore(dbClient);
-        const expectedPathsCache = new Cache<[TestSpecByPath, string | undefined], string>(getExpectedCacheKey);
-
-        const imagesInfoSaver = new ImagesInfoSaver({
-            imageFileSaver: htmlReporter.imagesSaver,
-            expectedPathsCache,
-            imageStore,
-            reportPath: htmlReporter.config.path
-        });
-
-        staticReportBuilder = StaticReportBuilder.create(htmlReporter, config, {dbClient, imagesInfoSaver});
-
-        handlingTestResults = Promise.all([
-            staticReportBuilder.saveStaticFiles(),
-            handleTestResults(hermione, staticReportBuilder)
-        ]).then(async () => {
-            await staticReportBuilder.finalize();
-        }).then(async () => {
-            await htmlReporter.emitAsync(htmlReporter.events.REPORT_SAVED, {reportPath: config.path});
-        });
-
-        htmlReporter.emit(htmlReporter.events.DATABASE_CREATED, dbClient.getRawConnection());
-    }));
-
-    hermione.on(hermione.events.RUNNER_START, withMiddleware((runner) => {
-        staticReportBuilder.registerWorkers(createWorkers(runner as unknown as CreateWorkersRunner));
-    }));
-
-    hermione.on(hermione.events.RUNNER_END, withMiddleware(async () => {
-        try {
-            await handlingTestResults;
-
-            logPathToHtmlReport(config);
-        } catch (e: unknown) {
-            logError(e as Error);
-        }
-    }));
-};
-
-async function handleTestResults(hermione: Hermione, reportBuilder: StaticReportBuilder): Promise<void> {
-    return new Promise((resolve, reject) => {
-        const queue = new PQueue({concurrency: os.cpus().length});
-        const promises: Promise<unknown>[] = [];
-
-        [
-            {eventName: hermione.events.TEST_PASS},
-            {eventName: hermione.events.RETRY},
-            {eventName: hermione.events.TEST_FAIL},
-            {eventName: hermione.events.TEST_PENDING}
-        ].forEach(({eventName}) => {
-            type AnyHermioneTestEvent = typeof hermione.events.TEST_PASS;
-
-            hermione.on(eventName as AnyHermioneTestEvent, (testResult: HermioneTestResult) => {
-                promises.push(queue.add(async () => {
-                    const formattedResult = formatTestResult(testResult, getStatus(eventName, hermione.events, testResult));
-
-                    await reportBuilder.addTestResult(formattedResult);
-                }).catch(reject));
-            });
-        });
-
-        hermione.on(hermione.events.RUNNER_END, () => {
-            return Promise.all(promises).then(() => resolve(), reject);
-        });
-    });
-}
+export = pluginHandler;
```

Comment:
Export default export of `./index.ts` as `module.exports`, because old versions of hermione do not support `export default`